lib/parseOwners.js

```diff
@@ -118,4 +123,7 @@ const arrayify = (val) => {
   return Array.isArray(val) ? val : [val]
 }
 
-module.exports = parseOwners
+module.exports = {
+  parseOwners: parseOwners,
+  checkIfPathMatches: checkIfPathMatches
```

Comment:
IMO It will be cleaner to extract this out particular generic function out to a "FileUtil" module. As it is now used in two places. Here and outside of this with a totally responsibility of work than parseOwners.

Reply:
this file does everything that has to do with parsing gitpattern, i don't think we need to refactor it out to a `fileUtils`, i'll rename the file to `gitpatternParser`

Reply:
fair enough.